<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>33</y>
      <width>168</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_password">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_JobId">
         <property name="text">
          <string>JobId</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_SysId">
         <property name="text">
          <string>SystemId</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_username"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_password"/>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="lineEdit_JobId"/>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_SysId"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCatalogs">
    <property name="title">
     <string>Catalogs</string>
    </property>
    <addaction name="actionBusbars"/>
    <addaction name="actionCableHeads"/>
    <addaction name="actionJumpers"/>
   </widget>
   <widget class="QMenu" name="menuPlacements">
    <property name="title">
     <string>Placements</string>
    </property>
    <addaction name="actionEquivalent_Source"/>
    <addaction name="actionBusbar_Section_Vertical"/>
    <addaction name="actionBusbar_Section_Horizontal"/>
    <addaction name="actionBusbar_Free_Style"/>
    <addaction name="separator"/>
    <addaction name="actionConnectivity_Node"/>
    <addaction name="actionCable_Had"/>
    <addaction name="actionSource_Sink_Connector"/>
    <addaction name="separator"/>
    <addaction name="actionJumper_z_0"/>
   </widget>
   <widget class="QMenu" name="menuLanguages">
    <property name="title">
     <string>Languages</string>
    </property>
    <addaction name="actionTurkish"/>
    <addaction name="actionBosnian"/>
   </widget>
   <widget class="QMenu" name="menuJobs">
    <property name="title">
     <string>Jobs</string>
    </property>
    <addaction name="actionSearch_Jobs"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Software"/>
   </widget>
   <addaction name="menuCatalogs"/>
   <addaction name="menuPlacements"/>
   <addaction name="menuJobs"/>
   <addaction name="menuLanguages"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBusbars">
   <property name="text">
    <string>Busbars</string>
   </property>
  </action>
  <action name="actionCableHeads">
   <property name="text">
    <string>CableHeads</string>
   </property>
  </action>
  <action name="actionJumpers">
   <property name="text">
    <string>Jumpers</string>
   </property>
  </action>
  <action name="actionTurkish">
   <property name="text">
    <string>Turkish</string>
   </property>
  </action>
  <action name="actionBosnian">
   <property name="text">
    <string>Bosnian</string>
   </property>
  </action>
  <action name="actionEquivalent_Source">
   <property name="text">
    <string>Equivalent Source</string>
   </property>
  </action>
  <action name="actionBusbar_Section_Vertical">
   <property name="text">
    <string>Busbar Section (Vertical)</string>
   </property>
  </action>
  <action name="actionBusbar_Section_Horizontal">
   <property name="text">
    <string>Busbar Section (Horizontal)</string>
   </property>
  </action>
  <action name="actionBusbar_Free_Style">
   <property name="text">
    <string>Busbar Free Style</string>
   </property>
  </action>
  <action name="actionConnectivity_Node">
   <property name="text">
    <string>Connectivity Node</string>
   </property>
  </action>
  <action name="actionCable_Had">
   <property name="text">
    <string>Cable Head</string>
   </property>
  </action>
  <action name="actionSource_Sink_Connector">
   <property name="text">
    <string>Source Sink Connector</string>
   </property>
  </action>
  <action name="actionJumper_z_0">
   <property name="text">
    <string>Jumper (z=0)</string>
   </property>
  </action>
  <action name="actionSearch_Jobs">
   <property name="text">
    <string>Search Jobs</string>
   </property>
  </action>
  <action name="actionAbout_Software">
   <property name="text">
    <string>About Software</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
